plugins/accelerated-peft/src/fms_acceleration_peft/gptqmodel/utils/peft.py

```diff
@@ -92,12 +92,12 @@ def find_all_linear_names(
 ):
     if not ignore:
         ignore = []
-    lm_head_name = model.lm_head_name
+    lm_head_name = model.lm_head
     if ignore_lm_head and lm_head_name not in ignore:
         ignore.append(lm_head_name)
     results = set()
     for n, m in model.named_modules():
-        if isinstance(m, torch.nn.Linear):
+        if isinstance(m, QuantLinearTriton):
```

Comment:
are we sure `QuantLinearTriton` is the only quant linear module that is used by gptqmodel?

Reply:
@fabianlim we limited gptqmodel to only QuantlinearTriton in this [function](https://github.com/foundation-model-stack/fms-acceleration/blob/32918eb94b23e18df256f45238f4f3bd988c5414/plugins/accelerated-peft/src/fms_acceleration_peft/gptqmodel/utils/model.py#L146) but i think setting `BaseQuantLinear` would more properly cover all gptqmodel qlinears modules.

Reply:
ok can we make the change then